### ISSUE TITLE
Unable to install dbt@0.19.0rc2 on Big Sur (Version 11.1)

### ISSUE THREAD

**Issue by kwigley** [fishtown-analytics/homebrew-dbt#10]
### Description

dbt fails to install with Homebrew. This occurs when Homebrew attempts to install `grpcio` with the following command:
```
==> /usr/local/Cellar/dbt@0.19.0-rc2/0.19.0rc2_1/libexec/bin/pip install -v --no-deps --no-binary :all: --no-user --ignore-installed /private/tmp/dbt-0.19.0-rc2--grpcio-20210117-98475-wurhff/grpcio-1.34.1
```

It produces the following error:
```
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/tmp/pip-req-build-gmlu06dy/setup.py", line 359, in <module>
        if mac_target and (pkg_resources.parse_version(mac_target) <
      File "/usr/local/Cellar/dbt@0.19.0-rc2/0.19.0rc2_1/libexec/lib/python3.8/site-packages/pkg_resources/__init__.py", line 128, in parse_version
        return packaging.version.Version(v)
      File "/usr/local/Cellar/dbt@0.19.0-rc2/0.19.0rc2_1/libexec/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/version.py", line 219, in __init__
        match = self._regex.search(version)
    TypeError: expected string or bytes-like object
```

<details>
<summary>Config info</summary>

```
==> Formula
Tap: fishtown-analytics/dbt
Path: /usr/local/Homebrew/Library/Taps/fishtown-analytics/homebrew-dbt/Formula/dbt@0.19.0-rc2.rb
==> Configuration
HOMEBREW_VERSION: 2.7.5-10-g8869208
ORIGIN: https://github.com/Homebrew/brew
HEAD: 8869208a601b7846f2e6597a49b4740cb3c06de2
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a9057054ff050df7d145ed2305dd68a0c1f2c7d6
Core tap last commit: 65 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: code
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit skylake
Clang: 12.0 build 1200
Git: 2.30.0 => /usr/local/bin/git
/usr/bin/curl --version
Curl: 7.64.1 => /usr/bin/curl
/usr/libexec/java_home --xml --failfast
/usr/libexec/java_home --xml --failfast
Java: 15.0.1
macOS: 11.1-x86_64
CLT: 12.3.0.0.1.1607026830
Xcode: 12.3
```
</details>

**Comment by kwigley** [comment]
This is the result of 2 things AFAIK.

1. we recently updated the dbt release process to use the latest version of pip to resolve and install dependencies (https://github.com/fishtown-analytics/dbt-release/pull/7), which does a better job of resolving external dependencies and added the `grpcio` dependency in 0.19.0rc2.
Pinned `google-api-core` dep in dbt-bigquery: https://github.com/fishtown-analytics/dbt/blob/dev/kiyoshi-kuromiya/plugins/bigquery/setup.py#L55
Pinned `google-api-core` dep in the google-cloud-bigquery dep which installs extra `grpcio` dep: https://github.com/googleapis/python-bigquery/blob/v2.3.1/setup.py#L32
2. The error has since been resolved but has not been released yet. https://github.com/grpc/grpc/commit/eba18503cc69c06aa021f940837da6c75d2273ac

I'm not sure the best solution here. We could remove `grpcio` from the formula or wait until `google-api-core` releases a new version. This same issue will probably affect subsequent versions of dbt.
